package.json

```diff
@@ -26,7 +26,7 @@
     "localforage": "1.7.1",
     "localforage-observable": "1.4.0",
     "marked": "github:mattermost/marked#ed33baecd7d7fa97d479ba22dde9d226b083d67d",
-    "mattermost-redux": "github:mattermost/mattermost-redux#6aee7c45bcde7d2079ee5b24f70e7e8d071ae0a6",
+    "mattermost-redux": "https://github.com/kennethjeremyau/mattermost-redux#gfycat",
```

Comment:
Fine for testing, but note that we'll need to update this to the actual latest commit in `mattermost-redux` once the other PR is merged.